test/org/parosproxy/paros/core/scanner/VariantJSONQueryTest.java

```diff
@@ -0,0 +1,75 @@
+package org.parosproxy.paros.core.scanner;
+
+import org.junit.Test;
+import org.parosproxy.paros.network.HttpMalformedHeaderException;
+import org.parosproxy.paros.network.HttpMessage;
+import org.parosproxy.paros.network.HttpRequestHeader;
+import org.zaproxy.zap.network.HttpRequestBody;
+
+import java.util.List;
+
+import static org.hamcrest.Matchers.empty;
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.assertThat;
+
+/**
+ * Unit test for {@link VariantJSONQuery}.
+ */
+public class VariantJSONQueryTest {
+    @Test
+    public void shouldHaveParametersListEmptyByDefault() {
+        // Given
+        VariantJSONQuery variantJSONQuery = new VariantJSONQuery();
+        // When
+        List<NameValuePair> parameters = variantJSONQuery.getParamList();
+        // Then
+        assertThat(parameters, is(empty()));
+    }
+
+    @Test
+    public void shouldFindStringValueInContent(){
```

Comment:
I'd be more explicit and say "Object" (or "JsonObject") instead of "Content" (for when the tests check an array).